Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Toy/boxes.yml

```diff
@@ -13,13 +13,24 @@
 # Boxes
 - type: entity
   id: BoxDonkSoftBox
-  name: "foamdart box"
+  name: "foam dart box"
```

Comment:
If we would want to be 100% consistent these should be named "ammunition box (foam dart)". Same goes for cap gun box. But considering these are toys, that might be overkill.

Reply:
Yeah my thinking was you'd get this as a box of foam darts from a crate or vending machine. It's not ammunition in the sense of the word but for code purposes it is obviously. Will leave this for maintainers to decide.